src/reanimated2/interpolation.js

```diff
@@ -51,3 +52,24 @@ export function interpolate(x, input, output, type) {
   }
   return internalInterpolate.apply({}, [x].concat(narrowedInput).concat(type));
 }
+
+export function interpolateColor(x, input, output, type) {
+  'worklet';
+
+  // Extrapolate.CLAMP is the default interpolation type for colors
+  type = type || Extrapolate.CLAMP;
+
+  const startColor = destructColor.apply({}, output[0]);
+  const endColor = destructColor.appy({}, output[1]);
```

Comment:
typo, I guess expected word is `apply`

Reply:
I wasn't able to test this patch (I hoped react-native-reanimated have some kind of CI tests to check pull requests).
Also I wasn't able to build react-native-reanimated npm package to test my patch manually https://github.com/software-mansion/react-native-reanimated/issues/1024 https://github.com/software-mansion/react-native-reanimated/issues/950
But I was able test code in isolation manually.

Reply:
Looks like with `appy` misspell it doesn't work. If I would able to build react-native-reanimated npm package from master branch I'll fix misspell and add tests.